concepts/type-conversion/about.md

```diff
@@ -1,3 +1,18 @@
-# About
+# About Type Conversion
```

Comment:
I need a second opinion: should we mention about converting to/from string specifically using the strconv package?

```go
var fortyTwo string = "42"
v, _ := strconv.Atoi(fortyTwo) // convert string to int
var i int = 42
str := strconv.Itoa(i) // convert int to string
```

Reply:
I would include the strconv package in this concept for now. It can then still be removed later if we want to make a separate exercise for it.

Reply:
Done